.circleci/config.yml

```
@@ -65,7 +65,7 @@ jobs:
     steps:
       - checkout
       - run: apt-get update -y
-      - run: apt-get install -y cmake gcc-mingw-w64-i686 g++-mingw-w64-i686 wget git sudo
+      - run: apt-get install -y cmake gcc-mingw-w64-i686 g++-mingw-w64-i686 wget git sudo zstd
```

```suggestion
      - run: apt-get install -y cmake gcc-mingw-w64-i686 g++-mingw-w64-i686 pkg-config-mingw-w64-i686 wget git sudo zstd
```

Looks like it's looking for pkg-config, I think it would be this package

---

Thanks, I've tried adding them in [`71249d2` (#1651)](https://github.com/diasurgical/devilutionX/pull/1651/commits/71249d233c52d45099c1cced1a1759eca9d26b07) but ultimately SDL_audiolib may need to be fixed to not require pkg-config https://github.com/realnc/SDL_audiolib/issues/7

---

Well it fixed that requirement, but now it's not finding "required package sdl2".

---

Strange, since `sdl2.pc` is there:

```
$ find . | grep pc                                                                   
./x86_64-w64-mingw32/lib/pkgconfig/sdl2.pc
./i686-w64-mingw32/lib/pkgconfig/sdl2.pc
```

---

We probably need to set the path to the pkgconfig binary

/usr/bin/i686-w64-mingw32-pkg-config
/usr/bin/x86_64-w64-mingw32-pkg-config

---

That worked but now it's:

```
--   Package 'flac', required by 'sndfile', not found
```

Looks like this sndfile is compiled with support for FLAC and OGG/Vorbis which means 3 more packages :(

---

All good now, hitting another issue but it's unrelated to this
https://github.com/realnc/SDL_audiolib/issues/8